docs/pages/setup/reference/backends.mdx

```diff
@@ -265,6 +265,10 @@ teleport:
 
     # This setting configures Teleport to save the recorded sessions in an S3 bucket:
     audit_sessions_uri: s3://Example_TELEPORT_S3_BUCKET/records
+
+    # Teleport by default stores stores audit events with an AWS TTL of 1 year.
+    # This value can be configured as shown below. If set to 0 seconds, TTL is disabled.
+    audit_retention_period: 60m
```

Comment:
@xacrimon Can you make this show the default, which should be 1 year.

Reply:
@xacrimon Once you make this change, ping me, I'll approve and we can merge this in.